frame/nomination-pools/src/lib.rs

```diff
@@ -1258,34 +1381,32 @@ pub mod pallet {
 		pub fn unbond_other(
 			origin: OriginFor<T>,
 			delegator_account: T::AccountId,
+			mut unbonding_points: BalanceOf<T>,
 		) -> DispatchResult {
 			let caller = ensure_signed(origin)?;
 			let (mut delegator, mut bonded_pool, mut reward_pool) =
 				Self::get_delegator_with_pools(&delegator_account)?;
-
 			bonded_pool.ok_to_unbond_other_with(&caller, &delegator_account, &delegator)?;
 
-			// Claim the the payout prior to unbonding. Once the user is unbonding their points
-			// no longer exist in the bonded pool and thus they can no longer claim their payouts.
-			// It is not strictly necessary to claim the rewards, but we do it here for UX.
+			unbonding_points = unbonding_points.min(delegator.active_points());
```

Comment:
I'm not strictly against it, but I don't like the min logic. I think we should strictly fail if the user attempts to many points. At the very least it should be an indicator to the user that their understanding of blockchain state is incorrect. But also reduces complexity

Reply:
leaving open for re-consideration.

Reply:
The more I think about it, the more I don't like the min logic - what is the justification?

Reply:
mainly to mimic staking, but will change it, not a problem.